packages/Menu/Menu.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <nav flex space-between align-center>
+      <div flex row space-between align-center>
+        <div>
+          <a>
+            <img :src="banner" alt />
+          </a>
+        </div>
+        <a @click="open = !open" class="burger" hidden-desktop>
+          <span v-if="open">&times;</span>
+          <span v-else>☰</span>
+        </a>
+      </div>
+      <div flex row space-between class="actions" :class="{ open }">
+        <ul flex row mobile-column align-center>
+          <li v-for="(category, i) in categories" :key="i" flex column>
+            <template v-for="item in items">
+              <slot v-if="category === item.key" :name="item.key" :item="item" />
+            </template>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Menu',
+  props: {
+    banner: { type: String, required: true },
```

Comment:
banner is not really clear, maybe logo would be better

Reply:
rename to `logo`